src/tex/appendix.tex

```diff
@@ -49,6 +49,86 @@ \subsection{Benchmark corpus}
     \script{plot_statistics.py}
 \end{figure}
 
+The corpus of the questions in Chembench, as shown in \Cref{tab:chembench_corpus}, can be divided according to which topic they belong, or which type of knowledge is required in order to solve them correctly.
+
+\begin{table*}[!t]
+    \centering
+        \caption{Summary of ChemBench question types.
+    }
+    \begin{tabular}{p{2.7cm}cp{10cm}}
+        \toprule
+        \textbf{Question Type} & \textbf{\%} & \textbf{Example Questions}  \\ 
+        \midrule
+        Chemical Preference & 35.0 & Imagine an early virtual screening campaign setting (accounting for simple aspects such as oral availability and small molecular profile, but no other modalities such as covalency or bifunctionality). Which of the following two candidates would you prefer for further development? \\
+        & & A. [START_SMILES]N#Cc1ccc(OCCCN2CC3CN(CCNS(=O)(=O)c4ccc(F)cc4)CC(C2)O3)cc1[END_SMILES] \\
+        & & B. [START_SMILES]O=C1CC(c2ccc(CC(NS(=O)(=O)c3cc(Cl)cc(Cl)c3)c3nc4ccccc4[nH]3)cc2)S(=O)(=O)N1[END_SMILES] \\
+        \midrule
+        Technical Chemistry & 1.5 & Which of the following statements is true about the different types of ideal reactors? \\
+        & & A. In a batch reactor, the composition is uniformly mixed and remains the same throughout the reactor and at the exit \\
+        & & B. In a batch reactor, the fluid passes through the reactor with no mixing of earlier and later entering fluid \\
+        & & C. In a mixed flow reactor, the composition changes with time but is uniform everywhere within the reactor \\
+        & & D. In a plug flow reactor, the fluid moves in single flow through the reactor with no mixing and no overtaking \\
+        \midrule
+        Organic Chemistry & 15.5 & What is the reaction mechanism that describes the following reaction (represented using reaction SMILES) [START_RXNSMILES]CCCl.CO[Na]>>[Na]Cl.CCOC[END_RXNSMILES]? \\
+        & & A. $E_1$ \\
+        & & B. $E_{cb}$ \\
+        & & C. $S_N1$ \\
+        & & D. $S_N2$ \\
+        \midrule
+        Toxicity/Safety & 24.2 & What details pertaining to a hazardous substance can be located on a Material Safety Data Sheet (MSDS)? \\
+        & & A. All hazards regarding the material \\
+        & & B. Physical and chemical properties \\
+        & & C. Precautions for safe handling and storage \\
```

Comment:
how did you pick examples? I feel we have more advanced / interesting examples for toxicity and safety

Reply:
By hand and by random election. I will change this one then